combirepo/rpm_patcher.py

```diff
@@ -353,8 +352,9 @@ def __deploy_qemu_package(self):
             self.__unpack_qemu_packages()
             qemu_executable_path = self.__find_qemu_executable()
 
-        binfmt.disable_all()
-        binfmt.register(self.architecture, qemu_executable_path)
+        combirepo_dir = os.path.abspath(os.path.dirname(__file__))
+        subprocess.call(["sudo", "python", os.path.join(combirepo_dir, "binfmt.py"),
```

Comment:
`/usr/bin/python` is not always compatible with our script. `python2` will be better.

Reply:
Done